FSQLocationBroker/FSQLocationBroker.m

```diff
@@ -514,17 +514,16 @@ - (void)refreshVisitSubscribers {
 - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations {
     
     BOOL isBackgrounded = applicationIsBackgrounded();
-    CLLocation *mostRecentLocation = [locations firstObject];
+    CLLocation *newestLocation = nil;
```

Comment:
We don't have an immediate benefit from this fix, and don't want to accidentally mess anything up.